pages/common/poetry.md

```diff
@@ -30,7 +30,3 @@
 - Bump the minor version of the project in `pyproject.toml`:
 
 `poetry version minor`
-
-- List all poetry subcommands:
-
-`poetry list`
```

Comment:
The documentation does say there exists a `poetry list` subcommand.
Reference: https://python-poetry.org/docs/cli/#list
Not sure if the docs are uptodate.

Reply:
When I run it locally it says `The command "list" is not defined.`

![image](https://user-images.githubusercontent.com/89593949/209611734-8c09647a-7185-40ec-9594-5566a2949067.png)

Reply:
> When I run it locally it says `The command "list" is not defined.`
> 
> ![image](https://user-images.githubusercontent.com/89593949/209611734-8c09647a-7185-40ec-9594-5566a2949067.png)

Yeah, this is known, and the command was removed upstream in favour of `poetry` alone to list all commands.
Ref: https://github.com/python-poetry/poetry/issues/1798, https://github.com/python-poetry/poetry/pull/1800, https://github.com/python-poetry/poetry/tree/master/src/poetry/console/commands

Reply:
Thank you, I will go to poetry github and submit a pr to remove it from their docs if I can. <3

Reply:
It seems someone already sent a PR for it actually, glad to have helped with tldr though :)